tasks/connectors/cobaltio/cobaltio.rb

```diff
@@ -0,0 +1,171 @@
+# frozen_string_literal: true
+
+require_relative "lib/cobaltio_helper"
+
+module Kenna
+  module Toolkit
+    class Cobaltio < Kenna::Toolkit::BaseTask
+      include Kenna::Toolkit::CobaltioHelper
+
+      SCANNER = "Cobalt.io"
+
+      def self.metadata
+        {
+          id: "cobaltio",
+          name: "Cobalt.io",
+          description: "Pulls findings from Cobalt.io",
+          options: [
+            { name: "cobalt_api_token",
+              type: "api_key",
+              required: true,
+              default: nil,
+              description: "Cobalt.io API token" },
+            { name: "cobalt_org_token",
+              type: "api_key",
+              required: true,
+              default: nil,
+              description: "Cobalt.io org token" },
+            { name: "retrieve_from",
+              type: "date",
+              required: false,
+              default: 90,
+              description: "default will be 90 days before today" },
+            { name: "projectName_strip_colon",
+              type: "boolean",
+              required: false,
+              default: false,
+              description: "strip colon and following data from Project Name - used as application identifier" },
+            { name: "packageManager_strip_colon",
+              type: "boolean",
+              required: false,
+              default: false,
+              description: "strip colon and following data from packageManager - used in asset file locator" },
+            { name: "package_strip_colon",
+              type: "boolean",
+              required: false,
+              default: false,
+              description: "strip colon and following data from package - used in asset file locator" },
```

Comment:
doesn't look like you need any of these strip colon params or am I missing something?

Reply:
You are right, I forgot to remove this after copying the metadata from Snyk. I will remove this.